Source/Shared/BPConfiguration.m

```diff
@@ -86,7 +86,7 @@
         "Print results in JUnit format."},
     {'F', "only-retry-failed", no_argument, "Off", BP_VALUE | BP_BOOL, "onlyRetryFailed",
         "If `failure-tolerance` is > 0, only retry tests that failed."},
-    {'l', "list-tests", no_argument, NULL, BP_VALUE, "listTestsOnly",
+    {'l', "list-tests", no_argument, NULL, BP_VALUE | BP_BOOL, "listTestsOnly",
```

Comment:
Actually, looking at the readme, I'm not certain what should be updated. It's already listed as a bool that defaults to false. I could add an "s" to "Only list tests in bundle" maybe?